host/xtest/pkcs11_1000.c

```diff
@@ -754,8 +823,21 @@ static void xtest_pkcs11_test_1003(ADBG_Case_t *c)
 	if (rv != CKR_OK)
 		goto out;
 
+	rv = test_set_pin(c, slot, CKU_USER);
+	if (rv != CKR_OK)
+		goto out;
 
+	rv = test_set_pin(c, slot, CKU_SO);
+	if (rv != CKR_OK)
+		goto out;
 
+	/*
+	 * CKU_CONTEXT_SPECIFIC is anything not CKU_USER or CKU_SO in order
+	 * to skip the initial login.
+	 */
+	rv = test_set_pin(c, slot, CKU_CONTEXT_SPECIFIC);
+	if (rv != CKR_OK)
+		goto out;
```

Comment:
add an empty line below. Or maybe remove lines 839 and 840.

Reply:
I'll remove this redundant test.